### PR TITLE
error for invalid expires-in and fix renew daemon

### PR DESCRIPTION
**Pull request by dopey**

### CODE REVIEW

**Comment by dopey** [comment]
@cookie-annihilator @tommy-56

File: command/ca/renew.go [hunk 1]
```diff
@@ -504,7 +513,7 @@ func (r *renewer) Daemon(outFile string, next, expiresIn, renewPeriod time.Durat
 				return nil
 			}
 		case <-time.After(next):
-			if _, err := r.RenewAndPrepareNext(outFile, expiresIn, renewPeriod); err != nil {
+			if next, err = r.RenewAndPrepareNext(outFile, expiresIn, renewPeriod); err != nil {
```
Comment (maraino) [inline command/ca/renew.go:516]: We're not using the next here for anything.

**Review by maraino — APPROVED** [review]
Looks good, but we can ignore the next inside the for loop.

File: command/ca/renew.go (hunk 1 above)
Comment (maraino) [reply command/ca/renew.go:516]: Ignore this, we are using in the channel.

**Pull request merged**